.github/workflows/codeql.yml

```diff
@@ -14,6 +14,10 @@ name: "CodeQL"
 on:
   push:
     branches: [ "main" ]
+    paths:
+      # '!**.md' will not work by itself to exclude *.md files. See https://github.com/orgs/community/discussions/25369
```

Comment:
Did you check this ?

Reply:
yep

Didn't test on codeql.yml, but tested using `!**.yml` on the main.yml workflow